lib/conjur/appliance/logging/sinatra.rb

```diff
@@ -1,16 +1,23 @@
 module Conjur
   module Appliance
     module Logging
-      # Don't log requests in the appliance. Nginx does this for us.
-      # Do always dump errors to STDERR.
       module Sinatra
-        def extended cls
+        def self.extended cls
           cls.module_eval do
-            enable :logging
-            set :dump_errors, true
-            
+
+            configure do
+              enable :logging
+              set :dump_errors, true
```

Comment:
The Doc for `dump_errors` says that the errors will be written to STDERR; I guess your fix for the runit logger will reroute those to STDOUT. Is that the plan?